client/components/status/status.js

```diff
@@ -42,13 +42,13 @@ export default class Status extends React.Component {
       userInfo: {},
       currentPage: 1,
       hasMoreSessions: false,
-      intervalId: null,
       screenWidth: window.innerWidth,
       loadSpinner: true,
       modalActive: false,
       rememberMe: false,
     };
     this.repeatLogin = false;
+    this.intervalId = null;
```

Comment:
is this really needed? it should work without this, please double check, if it works, we can omit it.